ha/monitor/k8s/monitor.py

```diff
@@ -51,36 +51,54 @@ def __init__(self, wait_time=10):
             _conf_stor_search = ConftStoreSearch()
 
             self.monitors = []
+            monitor_args = {}
 
             # event output in pretty format
-            kwargs = {K8SClientConst.PRETTY : True}
+            monitor_args['watch_args'] = {K8SClientConst.PRETTY : True}
 
-            # Seting a timeout value, 'timout_seconds', for the stream.
+            # Setting a timeout value, 'timeout_seconds', for the stream.
             # timeout value for connection to the server
             # If do not set then we will not able to stop immediately,
-            # becuase synchronus function watch.stream() will not come back
+            # because synchronous function watch.stream() will not come back
             # until catch any event on which it is waiting.
-            kwargs[K8SClientConst.TIMEOUT_SECONDS] = K8SClientConst.VAL_WATCH_TIMEOUT_DEFAULT
+            monitor_args['watch_args'][K8SClientConst.TIMEOUT_SECONDS] = wait_time
```

Comment:
This timeout was 5 earlier and now its changed to 10.
Any particular reason ?

Reply:
hey not changed, I just corrected as I am using const directly here instead of sending from constructor input.
It is still 5, the value you are looking at is default value which is not getting used as input is given

Reply:
okay